cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/runtime/artifact/ArtifactStoreTest.java

```diff
@@ -84,7 +87,10 @@
 
   @BeforeClass
   public static void setup() throws Exception {
-    artifactStore = AppFabricTestHelper.getInjector().getInstance(ArtifactStore.class);
+    CConfiguration cConf = CConfiguration.create();
+    // any plugin which requires transaction will be excluded
+    cConf.set(Constants.Plugin.EXCLUDE_WITH_REQUIREMENTS, Requirements.TRANSACTIONS);
```

Comment:
Should test with multiple values in the exclude.

Reply:
Good point.